lib/husky_musher/templates/registration_required.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h2>{% block title %}Registration Required{% endblock %}</h2>
+{% if not redcap_record_exists %}
+    <p>
+        <b>{{ netid }}</b> has not signed up!
+    </p>
+{% elif not registration_complete %}
+    <p>
+        <b>{{ netid }}</b> has not finished signing up for Husky Coronavirus Testing!
+    </p>
+{% endif %}
```

Comment:
Could this just be simplified to an else block, and remove registration_complete as a parameter for the template? It looks like this template only gets called in the case that registration is not complete.

Reply:
Yep! That was leftover from a previous iteration. I switched to an else block in a new commit.